require.js

```diff
@@ -1236,7 +1236,8 @@ var requirejs, require, define;
             while (defQueue.length) {
                 args = defQueue.shift();
                 if (args[0] === null) {
-                    return onError(makeError('mismatch', 'Mismatched anonymous define() module: ' + args[args.length - 1]));
+                    var module = args[args.length - 1];
+                    return onError(makeError('mismatch', 'Mismatched anonymous define() module: ' + module));
```

Comment:
I would prefer a different name than `module` for this local variable, I like to reserve that for the special module ID variable. And since this version of requirejs has not switched to locally defined variables next to their first use, then I think just doing a line break after the + is enough:

```javascript
                    return onError(makeError('mismatch', 'Mismatched anonymous define() module: ' + 
                                   args[args.length - 1]));

```